src/data/foldgroups.txt

```diff
@@ -26,5 +26,5 @@
 0	Jarlsberg's pan (Cosmic portal mode)	Jarlsberg's pan
 0	Sneaky Pete's leather jacket	Sneaky Pete's leather jacket (collar popped)
 0	toggle switch (Bounce)	toggle switch (Bartend)
-0	January's Garbage Tote	deceased crimbo tree	broken champagne bottle	tinsel tights	wad of used tape	makeshift garbage shirt
+0	January's Garbage Tote	deceased crimbo tree	broken champagne bottle	tinsel tights	wad of used tape	makeshift garbage shirt	replica January's Garbage Tote
```

Comment:
I have to ask: does this mean that Kolmafia now believes you can fold an item into a replica January's Garbage Tote?

Do we want to prevent that attempt?

Reply:
It does, but I do not know why the user would do this. The replica tote does not do anything, and I can only see it occurring if:
* a user asks whether the replica tote is creatable when they have a normal tote
* a user tries to fold something into the replica tote

So I don't feel like it's something that would come up in normal play. Of course, I may be proven wrong ;)

Reply:
Oh, I agree. It's not useful behavior - since it doesn't actually work - but users are full of surprises. ;)